install_sudo.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure I understand everything that is happening here, but can't this be travis-cached, instead of being cleaned and re-done each time?

Reply:
it's just a hacky experiment, trying to get as far as I can while minimizing the changes

Reply:
the file comes from https://github.com/imachug/win-sudo/blob/master/install.sh